src/nyc_trees/gulpfile.js

```diff
@@ -199,10 +200,19 @@ function browserifyTask(bundler) {
 }
 
 gulp.task('sass', ['clean'], function() {
-    return gulp.src('sass/main.scss')
-        .pipe(sourcemaps.init())
-        .pipe(sass({style: 'compressed'}))
-        .pipe(sourcemaps.write('./'))
+    var autoprefixProcessor = autoprefixer({ browsers: ['> 1% in US'] });
+
+    return sass('sass/main.scss', { sourcemap: true })
+        .on('error', function (err) {
+            console.error('Sass error', err.message);
```

Comment:
Do you know if we exit non-zero if there is a sass error? A failure in sass compilation should fail the build.